### ISSUE TITLE
MemoryEditor asserts in ErrorCheckUsingSetCursorPosToExtendParentBoundaries

### ISSUE THREAD

**Issue by learn-more** [ocornut/imgui_club#34]
When using `MemoryEditor` as last item inside an existing window, with `edit.OptShowOptions = false;` and `#define IMGUI_DISABLE_OBSOLETE_FUNCTIONS` it asserts (on `Code uses SetCursorPos()/SetCursorScreenPos() to extend window/parent boundaries. Please submit an item e.g. Dummy() to validate extent.`).

Version: `v1.90.3-docking`
Backend: `imgui_impl_glfw.cpp + imgui_impl_vulkan.cpp`
Compiler/OS: `Win10 22H2 + MSVC 2022 (17.7.4)`
Full config/build information:
```
Dear ImGui 1.90.3 (19030)
--------------------------------
sizeof(size_t): 4, sizeof(ImDrawIdx): 2, sizeof(ImDrawVert): 20
define: __cplusplus=199711
define: IMGUI_DISABLE_OBSOLETE_FUNCTIONS
define: IMGUI_DISABLE_OBSOLETE_KEYIO
define: _WIN32
define: _MSC_VER=1900
define: _MSVC_LANG=201402
define: IMGUI_HAS_VIEWPORT
define: IMGUI_HAS_DOCK
--------------------------------
io.BackendPlatformName: imgui_impl_glfw
io.BackendRendererName: imgui_impl_vulkan
io.ConfigFlags: 0x00000443
 NavEnableKeyboard
 NavEnableGamepad
 DockingEnable
 ViewportsEnable
io.ConfigViewportsNoDecoration
io.ConfigInputTextCursorBlink
io.ConfigWindowsResizeFromEdges
io.ConfigMemoryCompactTimer = 60.0
io.BackendFlags: 0x0000140E
 HasMouseCursors
 HasSetMousePos
 PlatformHasViewports
 RendererHasVtxOffset
 RendererHasViewports
--------------------------------
io.Fonts: 1 fonts, Flags: 0x00000000, TexSize: 512,64
io.DisplaySize: 1280.00,720.00
io.DisplayFramebufferScale: 1.00,1.00
--------------------------------
style.WindowPadding: 8.00,8.00
style.WindowBorderSize: 1.00
style.FramePadding: 4.00,3.00
style.FrameRounding: 0.00
style.FrameBorderSize: 0.00
style.ItemSpacing: 8.00,4.00
style.ItemInnerSpacing: 4.00,4.00
```

To reproduce:
* Download `imgui_memory_editor.h` from [ea49dd3](https://github.com/ocornut/imgui_club/commit/ea49dd3c6803088d50b496e3fe981501543b7cbc) and drop it next to `imgui.h`
* Enable the attached patch
* Open `Another Window`

Patch :
```patch
$ git diff
diff --git a/examples/example_glfw_vulkan/main.cpp b/examples/example_glfw_vulkan/main.cpp
index c852e5f4d..ad5227295 100644
--- a/examples/example_glfw_vulkan/main.cpp
+++ b/examples/example_glfw_vulkan/main.cpp
@@ -16,6 +16,7 @@
 #include "imgui.h"
 #include "imgui_impl_glfw.h"
 #include "imgui_impl_vulkan.h"
+#include "imgui_memory_editor.h"
 #include <stdio.h>          // printf, fprintf
 #include <stdlib.h>         // abort
 #define GLFW_INCLUDE_NONE
@@ -535,10 +536,15 @@ int main(int, char**)
         // 3. Show another simple window.
         if (show_another_window)
         {
+            static MemoryEditor edit;
+            static unsigned char data[100];
+            edit.OptShowOptions = false;
+
             ImGui::Begin("Another Window", &show_another_window);   // Pass a pointer to our bool variable (the window will have a closing button that will clear the bool when clicked)
             ImGui::Text("Hello from another window!");
             if (ImGui::Button("Close Me"))
                 show_another_window = false;
+            edit.DrawContents(data, sizeof data, 0);
             ImGui::End();
         }

diff --git a/imconfig.h b/imconfig.h
index d556cbaf1..13166d820 100644
--- a/imconfig.h
+++ b/imconfig.h
@@ -27,7 +27,7 @@
 //#define IMGUI_API __declspec( dllimport )

 //---- Don't define obsolete functions/enums/behaviors. Consider enabling from time to time after updating to clean your code of obsolete function/names.
-//#define IMGUI_DISABLE_OBSOLETE_FUNCTIONS
+#define IMGUI_DISABLE_OBSOLETE_FUNCTIONS
 //#define IMGUI_DISABLE_OBSOLETE_KEYIO                      // 1.87+ disable legacy io.KeyMap[]+io.KeysDown[] in favor io.AddKeyEvent(). This is automatically done by IMGUI_DISABLE_OBSOLETE_FUNCTIONS.

 //---- Disable all of Dear ImGui or don't implement standard windows/tools.
```

**Comment by ocornut** [comment]
Thank you for reporting. This is now fixed with d93fbf853

**Comment by learn-more** [comment]
> Thank you for reporting. This is now fixed with [d93fbf8](https://github.com/ocornut/imgui_club/commit/d93fbf853c5648b5c1404b8595b5ab843dd0a74c)

Thanks for the fast fix!
